about.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <title>Prog Rock Revival Concert</title>
+</head>
+<body>
+  <header>
+		<a class="header-icon" href="#">
+			<i class="fa-brands fa-facebook-f"></i>
+		</a>
+    <a class="header-icon" href="#">
+			<i class="fa-brands fa-instagram"></i>
+		</a>
+    <a class="header-icon" href="#">
+			<i class="fa-brands fa-twitter"></i>
+		</a>
+    <a class="header-icon" href="#">
+			<p>English</p>
+		</a>
+  </header>
+  <nav>
+    <a class="hamb-icons" id="menuHamburger">
+			<i id="hamburgerIcon" class="fa-solid fa-bars"></i>
+			<i id="closeIcon" class="fa-solid fa-xmark"></i>
+		</a>
+    <div class="prr-logo">
+      <a href="index.html">
+        <img src="./imgs/prog-rock-rev-logo.png" alt="prog-rock-revival logo">
+      </a>
+    </div>
+    <div class="nav-items" id="items-menul">
+      <ul class="ul-menu">
+        <li>
+          <a href="about.html" id="about-link">About</a>
+        </li>
+        <li>
+          <a href="index.html#schedule" class="menuItem">Schedule</a>
+        </li>
+        <li>
+          <a href="index.html#artists" class="menuItem">Artists</a>
+        </li>
+        <li>
+          <a href="index.html#sponsors" class="menuItem">Sponsors</a>
+        </li>
+        <li>
+          <a href="#" class="buy-tickets" id="buy-link"> Buy tickets! </a>
```

Comment:
- [ ] According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Colors, Typography, and Layout. So kindly, on the mobile version, you should update/change the style of the SEE THE WHOLE PROGRAM Link to look similar to the original design 💯👌

| Your Design  | Design Template |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/181866484-07858161-1dfc-42b8-b02c-d7cc66c1718b.png)  | ![template](https://user-images.githubusercontent.com/52934128/181866400-825250a6-b568-4122-b4f9-4b25e6aaeb4f.png) |